OrbitClientProtos/capture_data.proto

```diff
@@ -56,7 +56,7 @@ message CallstackInfo {
 
 message LinuxAddressInfo {
   uint64 absolute_address = 1;
-  string module_name = 2;
+  string module_name = 2; // This is actually the full module path
```

Comment:
should we rename it?

Reply:
I am in favor to rename it. This breaks compatibility between saved captures, correct? I am going to ask in chat if we already broke it. if yes, I will change it.